### ISSUE TITLE
[Suggestion] Monika talk about genre of music that you listen to

### ISSUE THREAD

**Issue by MetalBrother** [Monika-After-Story/MonikaModDev#4920]
Once, Monika said that:" If you blasting some good ol' rock n' roll, I'm gladly listen to it with you", or something like that. So I load some rock'n'roll songs but she don't have any reaction at all. So I think it would be great if Monika will able to react to music genre of song that playing. (At least reaction to genre about player talked with her). It may be a reaction that happens only once, because it will annoying, hear the same thing over and over again.

**Comment by yurireeka** [comment]
That'll be hilarious if she could see the waveforms of songs and bump her head to all high-octane guitar solos.

**Comment by lunulae** [comment]
I could see this potentially working for those who have the genre field filled out in the properties of their music. Rather than actually "listening" to the song she could read the file to find out what kind of music's currently playing. Her responses would be pretty general, and more obscure genres might not have a unique reaction, but it could be a nice little addition.

Rather than have a random dialogue, I think it would be a good idea to have a question in the Music category, "Hey, Monika... What do you think of this song?"

**Comment by Dominic18** [comment]
So what kind of music does she like?

**Comment by lunulae** [comment]
From memory I believe she has topics about enjoying rock, rap, and jazz so far, so the responses could riff on those a little or prompt those topics.

I'm sure she'd have something insightful to say about most genres though.

**Comment by MetalBrother** [comment]
> I could see this potentially working for those who have the genre field filled out in the properties of their music. Rather than actually "listening" to the song she could read the file to find out what kind of music's currently playing. Her responses would be pretty general, and more obscure genres might not have a unique reaction, but it could be a nice little addition.

Rather than have a random dialogue, I think it would be a good idea to have a question in the Music category, "Hey, Monika... What do you think of this song?

Exactly what I thought

**Comment by MetalBrother** [comment]
-she will defenitly not ready to a straight hour and half of death metal-

**Comment by MetalBrother** [comment]
**And about 2 hours of thrash metal**

**Comment by Rai99** [comment]
#1953

It's supposedly possible for us to detect what genre some songs are via the genre tag, but one thing to consider would be that not all music has that tag, and might not be properly labeled anyway, but that currently sits at low priority.

**Comment by Dominic18** [comment]
I wonder what kind of music the other three would lile

**Comment by MetalBrother** [comment]
> #1953
> 
> It's supposedly possible for us to detect what genre some songs are via the genre tag, but one thing to consider would be that not all music has that tag, and might not be properly labeled anyway, but that currently sits at low priority.

Understandable, but I think that would be a nice feature, so hope one day it will be done.

**Comment by MetalBrother** [comment]
> I wonder what kind of music the other three would like

Well, obviously Natsuki listen to k-pop or j-pop, or some vocaloids, Yuri can listen darkwave, maybe embient, or something since she likes horrors. Sayori is hard to tell, cause she mentally unstable, so I think she listen from pop to I don't know emo or goth music? Something like that.

**Comment by ThePotatoGuy** [comment]
[closing as duplicate]
